src/TetherContent.js

```diff
@@ -28,8 +28,9 @@ class TetherContent extends React.Component {
     super(props);
 
     this.handleDocumentClick = this.handleDocumentClick.bind(this);
-    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
     this.toggle = this.toggle.bind(this);
+    this.checkTogglerTarget = this.checkTogglerTarget.bind(this);
```

Comment:
They shouldn't need to be bound. Maybe there is a way to fix the test instead of changing the code?